pkg/localize/locales/en/cmd/kafka_update.en.toml

```diff
@@ -36,30 +39,49 @@ one = 'Name of the Kafka instance you want to update'
 description = 'Description for the --owner flag'
 one = 'ID of the user you want to set as the owner of this Kafka instance'
 
+[kafka.update.flag.reauthentication]
+one = 'Enable connection reauthentication for the Kafka instance'
+
 [kafka.update.flag.yes]
 one = 'Forcibly update the Kafka instance without confirmation'
 
 [kafka.update.summaryTitle]
 one = 'Summary of Proposed Changes'
 
+[kafka.update.reauthentication.disclaimer]
+one = '''
+A suspected incident of credential leak for a Kafka instance with disabled
```

Comment:
This looks different than what doc had :)
Should I correct this one?

Reply:
The reviewed para looked too long, tried to combine  best bits from that and the other para without losing critical info.

Reply:
It lost meaning and form IMHO.
We need to inform user about side effect - 
this one has number of issues etc.